packages/paypal-commerce-integration/src/paypal-commerce-venmo/paypal-commerce-venmo-button-strategy.ts

```diff
@@ -110,7 +111,9 @@ export default class PayPalCommerceVenmoButtonStrategy implements CheckoutButton
 
         const buttonRenderOptions: PayPalCommerceButtonsOptions = {
             fundingSource,
-            style: this.paypalCommerceIntegrationService.getValidButtonStyle(style),
+            style: this.getValidVenmoButtonStyles(
+                this.paypalCommerceIntegrationService.getValidButtonStyle(style),
+            ),
```

Comment:
The code can be simplified by moving `this.paypalCommerceIntegrationService.getValidButtonStyle` method usage inside `getValidVenmoButtonStyles` method:

```suggestion
            style: this.getValidVenmoButtonStyles(style),
```

Reply:
fixed